executor/executor.cc

```diff
@@ -431,6 +430,7 @@ int main(int argc, char** argv)
 		}
 	}
 
+	df_enable();
```

Comment:
Covering all cases is a bit more complex.
Sandboxing code below forks the process before calling loop(), so for sandboxed programs this call won't have any effect.
I suggest moving it to execute_one(), which is already executed by the proper process.